lighthouse-core/formatters/table.js

```diff
@@ -78,13 +78,18 @@ class Table extends Formatter {
 
     const rows = results.map(result => {
       const cols = headingKeys.map(key => {
+        let value = result[key];
+        if (typeof value === 'undefined') {
```

Comment:
Sounds like we might need some shared helps for the code and empty cell formatting. Should we do that here or consolidate later?